bibliopixel/drivers/driver_test.py

```diff
@@ -0,0 +1,90 @@
+import unittest
```

Comment:
No tests in the main package. Let's put anything like this in a root level tests folder.